File: app/src/main/java/fr/nspu/dev/recordvoicelearning/model/Peer.java
package fr.nspu.dev.recordvoicelearning.model;

/**
 * Created by nspu on 18-02-03.
 */

public interface Peer extends CollectionObject {
     int getFolderId();
     String getFileNameQuestion();
     String getFileNameAnswer();
     int getCount();
     int getKnowledge();
}

File: app/src/main/java/fr/nspu/dev/recordvoicelearning/model/Peer.kt
package fr.nspu.dev.recordvoicelearning.model

/**
 * Created by nspu on 18-02-03.
 */

interface Peer : CollectionObject {
    val folderId: Int?
    val fileNameQuestion: String?
    val fileNameAnswer: String?
    val count: Int?
    val knowledge: Int?
}
